src/_pytest/pytester.py

```diff
@@ -33,12 +33,18 @@
 from _pytest.main import ExitCode
 from _pytest.main import Session
 from _pytest.monkeypatch import MonkeyPatch
+from _pytest.nodes import Collector
+from _pytest.nodes import Item
 from _pytest.pathlib import Path
+from _pytest.python import Module
 from _pytest.reports import TestReport
+from _pytest.tmpdir import TempdirFactory
 
 if TYPE_CHECKING:
     from typing import Type
 
+    _PluggyPlugin = object
+
```

Comment:
Also used in https://github.com/pytest-dev/pytest/pull/6449/files#r371502971 - where to put it?